cram_pr2/cram_pr2_shopping_demo/src/setup.lisp

```diff
@@ -188,22 +188,33 @@
          (assert (btr:joint-state ?world ?robot (("torso_lift_joint" 0.15d0)))))))
 
 (defun spawn-basket ()
-  (btr:add-object btr:*current-bullet-world* :basket
-                  :b
-                  (cl-transforms:make-pose
-                   (cl-transforms:make-3d-vector 0.8 0.2 0.75)
-                   (cl-transforms:make-quaternion 0 0 0 1))
-                  :mass 1
-                  :length 0.5 :width 0.3 :height 0.18 :handle-height 0.09)
-  (let ((basket-desig (desig:an object (type basket) (name b))))
-    (coe:on-event
-     (make-instance 'cpoe:object-attached-robot
-       :link "l_wrist_roll_link"
-       :not-loose t
-       :grasp :front
-       :arm :left
-       :object-name :b
-       :object-designator basket-desig))))
+   (urdf-proj:with-simulated-robot
```

Comment:
This will not work because the function is called at startup of a node, and you cannot perform stuff at this point yet...